README.md

```diff
@@ -35,6 +35,25 @@ through the Docker Compose
 docker-compose build
 ```
 
+### Authentication
+
+You will need to create a private and public key and add to your environment variables.
+
+First create the private and public keys:
+
+```shell
+openssl genrsa -out private_key.pem 2048
+openssl rsa -pubout -in private_key.pem -out public_key.pem
+```
+
+You will then need to base64 encode the private and public keys to use in env variables:
+
+```shell
+# copy each of the keys then in OSX:
+pbpaste | base64 | pbcopy
+# add to 
```

Comment:
Was there any text to be added to the '#add to' ?
Also, might be worth just saying to base64 encode before adding to your . env file as the 'pbpaste | base64 | pbcopy' line wasn't very clear i.e  once typed that command, what does the user do? I had to lookup what pbpaste/pbcopy did

Reply:
good point, it should read - #paste copied base64 string into appropriate env variables